src/Bicep.Core.Samples/Files/TestFramework_CRLF/main.diagnostics.bicep

```diff
@@ -14,13 +14,15 @@ test testMain 'samples/main.bicep' = {
 }
 
 test testMain2 'samples/main.bicep' = {  
+//@[05:14) [BCP028 (Error)] Identifier "testMain2" is declared multiple times. Remove or rename the duplicates. (CodeDescription: none) |testMain2|

```

Comment:
You want this baseline to compile, so it would be best to rename the test symbols so that each has a unique name.